reflectable/lib/mirrors.dart

```diff
@@ -489,7 +489,7 @@ abstract class InstanceMirror implements ObjectMirror {
   ///     }
   ///
   /// Required capabilities: [delegate] requires the [delegateCapability].
-  dynamic delegate(Invocation invocation);
+  Object? delegate(Invocation invocation);
```

Comment:
There's a typo in the comment above: `invocation.positionArguments[0]);` should be `invocation.positionalArguments[0]);`

Reply:
Thx!